src/app/core/parameters/pointArray.ts

```diff
@@ -1,12 +1,13 @@
 import Parameter from "../parameter";
 import Registry from "../registry";
 import pointValue from "./pointValue";
+import {Segment} from "@/app/core/init"
 
 export default class PointArray extends Parameter {
     static readonly typeString: string = "PointArray";
     static readonly description: string = "Should be an array of points";
 
-    constructor(value: Array<Array<number>>) {
+    constructor(value: Segment) {
```

Comment:
Theres another comment of mine where I say you need to use `Array<Point>` use segment instead